### PR TITLE
ARROW-6304: [Java][Doc] Add a description to each module

### PR DESCRIPTION
**Pull request by emkornfield**
* Note experimental and contrib modules as the first part of the description.

### CODE REVIEW

File: java/performance/pom.xml [hunk 1]
```diff
@@ -19,6 +19,7 @@
     <artifactId>arrow-performance</artifactId>
     <packaging>jar</packaging>
     <name>Arrow Performance Benchmarks</name>
+    <description>JMH Performance benchmarks for other Arrow lirbaries.</description>
```
Comment (liyafan82) [inline java/performance/pom.xml]: lirbaries -> libraries

Comment (emkornfield) [reply java/performance/pom.xml]: thanks fixed.

**Comment by emkornfield** [comment]
@pravindra mind taking a look?

**Review by liyafan82 — APPROVED** [review]
LGTM. Thanks for doing this.

File: java/plasma/pom.xml [hunk 2]
```diff
@@ -18,6 +18,7 @@
     </parent>
     <artifactId>arrow-plasma</artifactId>
     <name>Arrow Plasma Client</name>
+    <description>Java client for the Plasma object store.</description>
```
Comment (jacques-n) [inline java/plasma/pom.xml]: I think this is experimental/contrib. It is still written completing ignoring other parts of arrow.

Comment (emkornfield) [reply java/plasma/pom.xml]: Revised.

**Pull request closed without merging**